docs/manual.rst

```diff
@@ -2007,30 +2007,45 @@ For example, here is an identity function applicable to both types of input
 
     g2 = transform(id_func, g1)
 
-A partially applied transform function from pyproj satisfies the requirements
-for `func`.
+
+If using `pyproj>=2.1.0`, the preferred method to project geometries is:
 
 .. code-block:: python
 
-    from shapely.ops import transform
-    from functools import partial
     import pyproj
 
-    proj_in = pyproj.Proj(init='epsg:4326')
-    proj_out = pyproj.Proj(init='epsg:26913')
+    from shapely.geometry import Point
+    from shapely.ops import transform
 
-    project = partial(
-        pyproj.transform,
-        proj_in,
-        proj_out)
+    wgs84_pt = Point(-72.2495, 43.886)
+
+    wgs84 = pyproj.CRS('epsg:4326')
+    utm = pyproj.CRS('epsg:32618')
```

Comment:
Can we capitalize EPSG in the two lines above @tomplex? CRS authorities are generally capitalized in GDAL and PROJ usage. In combination with "init", on the other hand, we do need to use lower case only because "epsg" is literally a filename.